media/transcoder/transcoder_test.go

```diff
@@ -37,6 +37,8 @@ const (
 	testBucketName           = "cloud-samples-data"
 	testBucketDirName        = "media/"
 	testVideoFileName        = "ChromeCast.mp4"
+	testConcatFileName1      = "ForBiggerEscapes.mp4"
+	testConcatFileName2      = "ForBiggerJoyrides.mp4"
```

Comment:
General testing question: why do we need separate files for this?

Reply:
In general, we don't need 2 separate files for this but I was trying to make the test the same as the HOWTO: https://cloud.google.com/transcoder/docs/how-to/concatenate-videos

I can concatenate the existing test file with itself if you think this is better.

Reply:
Refactored to only include one additional file. Unfortunately, the API will not let you concat a file with itself so one more file is needed.